jwst/datamodels/mrsxartcor.py

```diff
@@ -0,0 +1,25 @@
+from .reference import ReferenceFileModel
+
+__all__ = ['MiriMrsXArtCorrModel']
+
+
+class MiriMrsXArtCorrModel(ReferenceFileModel):
```

Comment:
Suggest name change to begin with just `Mir` to be `MirMrsXArtCorr`. This is then consistent with other MIRI-specific models.

Reply:
Thought I do note there is an `MiriIFUCubeParsModel`, but others, such as `MirLrsPhotomModel` have just the three letters.

Reply:
I actually see a mixture of "Mir", "Miri", and "MIRI" in use in various datamodel class names, but I think the trend of the more recent ones is in the direction of "Mir".